src/GithubApi.php

```diff
@@ -142,7 +142,7 @@ public function getPullRequestForCommit(string $commit, string $repo): ?array
         $bestItem = null;
         foreach ($data['items'] as $item) {
             // make sure the PR referenced isn't from a different repository
-            if (false === strpos($item['html_url'], sprintf('%s/pull', $repositoryName))) {
+            if (!str_contains($item['html_url'], sprintf('%s/pull', $repositoryName))) {
```

Comment:
Can you revert the 2 unrelated changes?

Reply:
I did. Had to commit this code to make fabbot.io happy in the first place.